lib/LaTeXML/Package/physics.sty.ltxml

```diff
@@ -242,11 +234,15 @@ Let('\cp',    '\crossproduct');
 DefMacro('\lx@physics@operator{}{}{}', sub {
     my ($gullet, $cs, $semantic, $function) = @_;
     my $cfunc = Invocation(T_CS('\@CSYMBOL'), $semantic);
-    my $size  = phys_readDelimiterSize($gullet);
-    my ($arg, $open, $close) = phys_readArg($gullet, 0, undef, undef, %physics_delimiters);
-    return ($arg
-      ? phys_applyFunction($cfunc, $function, $size, $open, $close, $arg)
-      : Invocation(T_CS('\DUAL'), undef, $cfunc, Invocation(T_CS('\@WRAP'), $function))); });
+    my $size  = phys_readSize($gullet);
+    my ($arg, $open, $close) = phys_readArg($gullet, 0, %physics_delimiters);
```

Comment:
seeing that the `phys_` namespace even has its own `readArg` method really suggests it should have a standalone perl class.

Reply:
probably would be a good thing, but then we've got to come up with a good idiom for importing everything from Package?

Reply:
Do we? How about just using `@EXPORT` as usual and doing:
```
use LaTeXML::Package::Physics;
```
from any `.ltxml` binding that needs those functions? And it can be `PhysicsSty` or `PhysicsAPI` or ... whichever name seems most appropriate.